### ISSUE TITLE
Game 'freezes' when unloading both elevators on same floor.

### ISSUE THREAD

**Issue by FourierTransformer** [eonarheim/BellTowerEscape#15]
This is a weird one. The game "freezes" if both elevators are on the same floor and the "stop" command has been sent (there may be more to it than this...). I have never been able to actually complete a game because of it.

The `stop` seems successful (and all Meeples transfer correctly):
```json
{"Success":true,"Message":"Stopped elevator 0 successfully"}
{"Success":true,"Message":"Stopped elevator 1 successfully"}
```
but from then on out, both elevators return:
```json
{"Success":false,"Message":"Elevator has already been moved"}
{"Success":false,"Message":"Elevator has already been moved"}
```

and then the status is always returned the same (including the "TimeUntilNextTurn")

**Comment by FourierTransformer** [comment]
OOOH boy, also happens when an enemy elevator stops on the same floor as you. Fun...

```
NEW TURN
TURN: 17	DELIVERED: 1
Enemy Elevator 0 is on floor 2 with 3 Meeples
Enemy Elevator 1 is on floor 9 with 2 Meeples
My Elevator 2 is on floor 9 with 3 Meeples
My Elevator 3 is on floor 5 with 3 Meeples
Stopped elevator 2 successfully
Moved elevator 3 down successfully

NEW TURN
TURN: 17	DELIVERED: 2
Enemy Elevator 0 is on floor 2 with 2 Meeples
Enemy Elevator 1 is on floor 9 with 5 Meeples
My Elevator 2 is on floor 9 with 2 Meeples
My Elevator 3 is on floor 4 with 3 Meeples
Elevator has already been moved
Elevator has already been moved
```

**Comment by FourierTransformer** [comment]
Resolved as of 94f82b9f15aa87075cfba035d292e362505e05b9
